examples/key.go

```diff
@@ -0,0 +1,35 @@
+package examples
+
+import (
+	crypto "github.com/libp2p/go-libp2p-crypto"
+)
+
+// KeyPair is a helper struct used to contain the parts of a key
+type KeyPair struct {
+	Private crypto.PrivKey
```

Comment:
FYI, PrivKey contains the public key as well (you can call `GetPublic()` on it to get the public part).

Reply:
Good point!